### PR TITLE
Adds new plants to botany

### PR DESCRIPTION
**Pull request by phil235**
(rice, oat, blue cherries, holy melon, parsnip, geranium, lily, snap corn, blumpkin, mimana, sweet potato, vanilla, red beet) and adding two new reagents (vanilla and rice).

Most of these are taken from https://tgstation13.org/phpBB/viewtopic.php?f=11&t=2435
Parsnip, sweet potato, oat, red beet and blue cherries don't have anything special.
Also fixes a small bug with russian revolver.

Thanks to @TZK13 , I <s>stole</s> used a lot of his code. :hear_no_evil:

### CODE REVIEW

**Comment by Chocobro** [comment]
sprites by me. once this is Definitely Merged i'll work on some food/drinks to give the plants more substance.

**Comment by pudl** [comment]
the orange poppy color swap should get an original sprite so it doesnt look shit

**Comment by Chocobro** [comment]
i don't know what you mean pudl please explain

**Comment by Miauw62** [comment]
rice is the best food, so i support this PR.

**Comment by Chocobro** [comment]
pretty sure this is ready to merge.

**Comment by Miauw62** [comment]
conflicts ;-;

**Comment by Chocobro** [comment]
>[23:48] (%phil235) Chocobro: ready to merge

*taps foot

**Comment by Chocobro** [comment]
great so my irc logs work with git markup that's just brilliant
phil said this is r2m also phone git can't delete comments

File: code/modules/hydroponics/growninedible.dm [hunk 1]
```diff
@@ -291,9 +298,36 @@
...
+	throw_range = 7
+	var/snap_pops = 1
+
+/obj/item/weapon/grown/snapcorn/add_juice()
+	..()
+	snap_pops = max(round(potency/8), 1)
+
+/obj/item/weapon/grown/snapcorn/attack_self(mob/user as mob)
+	..()
+	user << "<span class='notice'>You pick up a snap pops from the cob.</span>"
```
Comment (Anonus) [inline code/modules/hydroponics/growninedible.dm]: >a snap pops

**Pull request merged**